bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=osp-director-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.13.1
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.13.0+git
```

Comment:
is this an intended change 1.13.1 -> 1.13.0?

Reply:
It's weird, but that is what it wants to put there.  I even have `1.13.1` installed:

```
# operator-sdk version
operator-sdk version: "v1.13.1", commit: "1659ab58a073999463e6dcdf18bdc359a315e091", kubernetes version: "1.21", go version: "go1.16.8", GOOS: "linux", GOARCH: "amd64"
```

I'm not sure why it generates `v1.13.0+git`.

Reply:
right, see this also in my latest patches. haven't realized before

Reply:
Is it because you installed from git that this is the case?

Reply:
I did indeed follow the standard operator-SDK installation docs for doing so from Github: https://sdk.operatorframework.io/docs/installation/#install-from-github-release

Reply:
Did I install from git source then? :)